ui/src/message_popup/mobile_prototype/texting_interface.jsx

```diff
@@ -0,0 +1,184 @@
+/* @flow weak */
+import React from 'react';
+import _ from 'lodash';
+import VelocityComponent from 'velocity-react/velocity-component';
+import 'velocity-animate/velocity.ui';
+
+import Divider from 'material-ui/Divider';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+
+import {StudentMessage, UserMessage, InfoMessage} from './message_components.jsx';
+
+export const TextBody = React.createClass({
```

Comment:
Would it work to split these two components into their own files?  If not please add a quick comment at the top of the file explaining what's in here.